client/src/main/resources/org/schlunzis/kurtama/client/fx/controller/login.fxml

```diff
@@ -49,16 +50,16 @@
                     <TextField fx:id="portField" GridPane.columnIndex="1" GridPane.rowIndex="1"/>
                     <Button text="i{login.button.connect}" onAction="#handleServerConnect" GridPane.columnIndex="0"
                             GridPane.rowIndex="2" GridPane.columnSpan="2"/>
-                    <HBox styleClass="progress" GridPane.columnIndex="0" GridPane.rowIndex="3" GridPane.columnSpan="2">
-                        <Region fx:id="progressIndicator"/>
-                        <Label fx:id="progressLabel" text="Connecting..."/>
-                    </HBox>
+                    <StatusView fx:id="statusView" GridPane.columnIndex="0" GridPane.rowIndex="3"
+                                GridPane.columnSpan="2"/>
                 </GridPane>
             </GridPane>
 
             <Label fx:id="versionLabel" AnchorPane.bottomAnchor="0" AnchorPane.rightAnchor="0"
                    styleClass="version-label"
                    text="Test"/>
+            <Button text="i{login.create.server}" AnchorPane.topAnchor="0" AnchorPane.rightAnchor="0"
```

Comment:
I'm still impressed, that this way of adding multi language support with generating the bindings really works. Well done!